components/game/screen.js

```diff
@@ -20,11 +20,13 @@ const blockShadowOpacity2 = "55";
 
 class GameScreen {
   constructor() {
-    this.canvas = document.getElementById("canvas");
+    this.speaker = new Speaker();
```

Comment:
Nit: How about `Audio/SoundPlayer` instead of `Speaker`??

My gut feeling this name is hardware speaker

Reply:
agree

Reply:
The 'Audio' is overlapped with audio element of html markup. And this module has few functions to call 'SoundPlayer'. So I named 'speaker'. Do you have a better name? @DevSDK

Reply:
How about `AudioPlayer` is much better.
Because the \<audio> indicates an audio resource that wants to play.
The name 'AudioPlayer' is meant intuitively to work as \<audio> resource player or controller.

Reply:
ok. I will change to `AudioPlayer`.